### ISSUE TITLE
bug: glyph onload

### ISSUE THREAD

**Issue by cxjay26** [jferragut/alkemy#56]
Glyph onload takes up whole view for split second. 

Possibly bug with library or styling.

**Comment by jferragut** [comment]
@cxjay26 
Found this: https://fontawesome.com/how-to-use/on-the-web/other-topics/server-side-rendering

I think this addresses the issue and we need to implement. I was loading like normal font awesome, but Gatsby is SSR. Upon looking further, I found that there are github issues that reference it as well.

https://github.com/FortAwesome/react-fontawesome/issues/134#issuecomment-421580721 
https://github.com/FortAwesome/react-fontawesome/issues/134#issuecomment-471940596

**To Fix:** 

- [ ] Below should be added to the layout component.

```javascript
import { config } from "@fortawesome/fontawesome-svg-core";
config.autoAddCss = false;
```

- [ ] Next add the following to the gatsby-browser.js file with the other CSS imports
```javascript
require("@fortawesome/fontawesome-svg-core/styles.css");
```

This merits testing, but should resolve the issue. 

**Explanation:**
With SSR, the Icons need to be pre-rendered and available for use. This means we need to disable font-awesome from generating the CSS after site loads and instead make it generate the css ahead of time.


**Acceptance Criteria:**
- [ ] All font awesome icons should load correctly
- [ ] When testing homepage, the icons should be normal sized on initial load (not huge as previously noted
- [ ] Tests pass locally
- [ ] Tests pass on build preview
